lib/src/Pos/Launcher/Configuration.hs

```diff
@@ -67,6 +72,40 @@ instance FromJSON Configuration where
 instance ToJSON Configuration where
     toJSON = genericToJSON defaultOptions
 
+data WalletConfiguration = WalletConfiguration
+    { ccThrottle :: !(Maybe ThrottleSettings)
+    } deriving (Show, Generic)
+
+defaultWalletConfiguration :: WalletConfiguration
+defaultWalletConfiguration = WalletConfiguration
+    { ccThrottle = Nothing
+    }
+
+instance FromJSON WalletConfiguration where
+    parseJSON = genericParseJSON defaultOptions
+
+instance ToJSON WalletConfiguration where
+    toJSON = genericToJSON defaultOptions
+
+data ThrottleSettings = ThrottleSettings
+    { tsRate   :: !Int
+    , tsPeriod :: !Int
```

Comment:
If this is "Number of requests" and "Amount of microseconds" - why do you use `Int` here? Maybe some `Word`-type?

Reply:
This is a good call, since we end up converting it to a `Word64` later anyway